backend/tslint.json

```diff
@@ -60,7 +61,30 @@
             "warn",
             "error"
         ],
-        "no-unused-variable": true
+        "no-unused-variable": true,
+        "completed-docs": [
+            true,
+            {
+                "enums": true,
+                "classes": true,
+                "functions": true,
+                "methods": true,
+                "properties": true,
+                "types":true,
```

Comment:
Du hast dir super viele Mühe gegeben, aber ich bin der Meinung das die JSDocs nur für classes, functions und methods verpflichtend sein sollte. Bei den anderen Typen erzeugt JSDoc sehr viele "Boilerplate" Kommentare. Ein gutes Beispiel ist z.B. das hier:

```
/**
* The helpText for a action
* @type{string}
*/
helpText: string;`
```
Oder:
```
/**
* A counter used for internal reasons
* @type {number}
*/
private static counter: number = 0;
```
Ich sehe in dieser Dokumentation keinen Vorteil. In beiden Fällen ist der Name sehr eindeutig und der Type kann auch direkt abgelesen werden. Warum daher JSDoc erzwingen?

Reply:
Weil es in anderen Fällen wiederum sinnvoll ist. Ich hatte einige Variablen und Properties, bei denen ich länger suchen musst, bis ich sie verstanden hatte. Diese waren bisher nicht kommentiert.  Das Problem ist wenn man es nicht generell erzwingt, dann werden diese auch weiterhin nicht kommentiert oder nur wenn man im PR darauf hinweist. Außerdem haben die durchgehend erzwungenen Kommentare, dass wir mit Abschluss eine komplette Dokumentation erzeugen können. Den Hauptfokus sehe ich aber eher bei oberem Punkt. Außerdem habe ich es in nur wenigen Stunden geschafft die ganzen JsDocs nachzuholen, so dass es aktiv während dem entwickeln wahrscheinlich nur ein paar Minuten pro Woche kostet. (Zumal man da nicht Backtracking machen muss, bis man alles versteht xD)